server/config.ts

```diff
@@ -60,6 +60,8 @@ let fastModelReasoning = false
 let slackHost = process.env.SLACK_HOST
 let VESPA_NAMESPACE = "my_content"
 let ragOffFeature = true
+let usePerUserServiceAccountSync = process.env.USE_PER_USER_SERVICE_ACCOUNT_SYNC === "true"
```

Comment:
this behavior should happen by default, one should require ENV only to fall back to legacy way